src/main/resources/static/_shared.yml

```diff
@@ -4568,6 +4588,15 @@ components:
         - FullPerson
         - RestrictedPerson
         - UnknownPerson
+    FullPersonSummary:
+      allOf:
+        - $ref: '#/components/schemas/PersonSummary'
+    PersonSummaryDiscriminator:
+      type: string
+      enum:
+        - FullPersonSummary
+        - RestrictedPersonSummary
+        - UnknownPersonSummary
```

Comment:
Like FullPersonSummary: I think we also need a RestrictedPersonSummary: and UnknownPersonSummary:. Neither of these need to provide additional properties e.g.
```
    RestrictedPersonSummary:
      allOf:
        - $ref: '#/components/schemas/PersonSummary'
    UnknownPersonSummary:
      allOf:
        - $ref: '#/components/schemas/PersonSummary'         
```